README.md

```diff
@@ -81,7 +81,7 @@ If you're new here, here are some things you should know:
 
 #### 💬 Join us on slack
 Please feel free to join us on Slack!
-You can sign up at https://rubyforgood.herokuapp.com and find us in #human-essentials.
+You can sign up at https://rubyforgood.org and find us in #human-essentials.
 
```

Comment:
Is there a reason we're not taking them directly to https://rubyforgood.slack.com/join/shared_invite/zt-1kfeimohe-KL~~~6Lkof7G94_7Ojd_Hw#/shared-invite/email  ?

Reply:
Mayyyybe? :grin: I figured whoever wrote the original doc wanted them to go to the page first (maybe to see more info about what RfG does?) Obviously I can change if you like.

Reply:
We can link to the page with the slack link here: https://rubyforgood.org/join-us or you can just put the invite link on (that is on that page) in the readme. Whichever you think is best :)

Reply:
Ok.. How about we do this...
1/  Use the direct link to the invite in the section on Slack

2/  Add a section below for Ruby for Good with text something like:

Ruby for Good    
Human Essentials is one of many projects initiated and run by Ruby for Good.   You can find out more about Ruby for Good at  https://rubyforgood.org   


It's plausible that it will redirect some people to other projects, but I'm ok with that if they are still Gooding.